ETC.tsv

```diff
@@ -13424,14 +13422,14 @@ ETC_20150717_013422	Social Function
 ETC_20150717_013423	Use the Social Function to interact with other players.
 ETC_20150717_013424	[Helper] You can leave your character to be used as Helpers when you logout. You need to be at least Rank 3 and iCoin is consumed.
 ETC_20150717_013425	Other players may hire you as Helpers by paying iCoins.
-ETC_20150717_013426	Helpers follows its host character and helps in combat.
-ETC_20150717_013427	[Friend] Use F7 key to open Friends window. Players can add each other to Friends List.
+ETC_20150717_013426	A Helper will follow the host character and assist in combat.
+ETC_20150717_013427	[Friend] Press the F7 key to open Friends window. Players can add each other to Friends List.
```

Comment:
[Friend] Press the F7 key to open Friends window. Players can add each other to Friends **->** [Friend] Press F7 to open Friends window. Players can add each other to the Friends List. 

For consistency. I personally like Buddy List better than Friends List. I have changed a few other lines related to this.